frontend/src/entities/configuration/activated-services-configuration.tsx

```diff
@@ -0,0 +1,81 @@
+/**
+ Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License").
+ You may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+import {
+    Alert,
+    Container, ContentLayout, Grid,
+    Header, SpaceBetween,
+    Toggle
+} from '@cloudscape-design/components';
+import React from 'react';
+
+const configurableServices = {
+    batchTranslate: 'Amazon Translate batch',
+    realtimeTranslate: 'Amazon Translate real-time',
+    emrCluster: 'Amazon EMR',
+    labelingJob: 'Amazon Ground Truth'
+};
+
+export type ActivatedServicesConfigurationProps = {
+    setFields: (any) => void;
```

Comment:
You don't have to change this here but since we use things like this `setFields` function all over the place maybe in the future we should have dedicated type definition we can use.

```typescript
type StringMap = {[key: string]: string};
```